assets/xml/objects/object_harfgibud.xml

```diff
@@ -58,11 +61,11 @@
         <Limb Name="gHarfgibudGibdoRightFootLimb" Type="Standard" EnumName="HARFGIBUD_GIBDO_LIMB_RIGHT_FOOT" Offset="0x852C" />
         <Skeleton Name="gHarfgibudGibdoSkel" Type="Flex" LimbType="Standard" LimbNone="HARFGIBUD_GIBDO_LIMB_NONE" LimbMax="HARFGIBUD_GIBDO_LIMB_MAX" EnumName="HarfgibudGibdoLimb" Offset="0x8580" />
 
-        <Animation Name="object_harfgibud_Anim_009D44" Offset="0x9D44" />
-        <Animation Name="object_harfgibud_Anim_00A164" Offset="0xA164" />
-        <Animation Name="object_harfgibud_Anim_00AE1C" Offset="0xAE1C" />
-        <TextureAnimation Name="object_harfgibud_Matanimheader_00AE50" Offset="0xAE50" />
-        <Animation Name="object_harfgibud_Anim_00B644" Offset="0xB644" />
+        <Animation Name="gHarfgibudLurchForwardAnim" Offset="0x9D44" />
+        <Animation Name="gHarfgibudRecoilFromHitAnim" Offset="0xA164" />
+        <Animation Name="gHarfgibudPanicAnim" Offset="0xAE1C" />
+        <TextureAnimation Name="gHarfgibudUnusedTexAnim" Offset="0xAE50" />
```

Comment:
Looking at the data, this looks to be another `EyeTexAnim` (notice the textures it uses are the eyes) that goes unused.